webpack-fallbacks-plugin.js

```diff
@@ -0,0 +1,22 @@
+function fallbacksPlugin(
+    context,
+    opts,
+) {
+    return {
+        name: 'webpack-fallbacks-plugin',
```

Comment:
Disabled this and saw this message. Include the blurb about the change here as a comment to help us remember why this is here.

```
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }
```